### ISSUE TITLE
Can't proceed with install - Not Authorized / Take ownership failed

### ISSUE THREAD

**Issue by alpiua** [Drive-Trust-Alliance/sedutil#380]
Hello. 
I've tried to install sedutil on my fresh arch linux installation 
Thinkpad x1 yoga gen3 / samsung MZVLB512HAJQ-000L7 / 5L2QEXA7

What i've done:
sedutil-cli --initialsetup debug /dev/nvme0n1
sedutil-cli --enablelockigrange 0 debug /dev/nvme0n1
sedutil-cli --setlockingrange debug /dev/nvme0n1

after last command my system had hang and from there I'm no able to access data.
Not a big deal, just a few hours to restore.
I can see the disk and probably can install a fresh OS on it.

But my concern is - when I'm booting from SED boot cd I have this:
linuxpba
...
Drive /dev/nvme0 SAMSUNG MZVLB512HAJQ-000L7      is OPAL NOT LOCKED

Trying to run any of these:
sedutil-cli --initialsetup debug /dev/nvme0n1
sedutil-cli --revertNoErase

I'm getting:
method status code NOT AUTHORIZED

So somehow I lost my keys. How really I do restore them ?

I've tryied clean every bios option related to TPM with no luck.
Thinkpad erase drive utility was not able even to load.

**Comment by r0m30** [comment]
If initial set up work the first time it won't work again, it relies on the MS ID being the password for the administrative SP. Did you use the password debug on the revertnoerase?￼

The Linux PBA says that the driver is not locked but you say that it can't be written to that doesn't make any sense to me. Are you using a non-US keyboard?

**Comment by alpiua** [comment]
I was able to write to it, but I couldn't revertnoerase. The password was "debug" for sure, on en_US keyboard layout.

Anyway, I was able to reset the device with 
sedutil-cli --yesIreallywanttoERASEALLmydatausingthePSID %PSID% /dev/nvme0

It would be helpful if you place this command in the instruction page on the webside, along with the explanation of what is PSID and were to get it.
It's not obvious that PSID is printed on the drive itself, it took some time to figure out things.
